WordPressAuthenticatorTests/Authenticator/WordPressAuthenticatorTests.swift

```diff
@@ -38,21 +45,352 @@ class WordPressAuthenticatorTests: XCTestCase {
         url = WordPressAuthenticator.baseSiteURL(string: punycode)
         XCTAssert(url == punycode)
     }
-
+    
     func testEmailAddressTokenHandling() {
         let email = "example@email.com"
         let loginFields = LoginFields()
         loginFields.username = email
         WordPressAuthenticator.storeLoginInfoForTokenAuth(loginFields)
-
+        
         var retrievedLoginFields = WordPressAuthenticator.retrieveLoginInfoForTokenAuth()
         var retrievedEmail = retrievedLoginFields.username
         XCTAssert(email == retrievedEmail, "The email retrived should match the email that was saved.")
-
+        
         WordPressAuthenticator.deleteLoginInfoForTokenAuth()
         retrievedLoginFields = WordPressAuthenticator.retrieveLoginInfoForTokenAuth()
         retrievedEmail = retrievedLoginFields.username
-
+        
         XCTAssert(email != retrievedEmail, "Saved loginFields should be deleted after calling deleteLoginInfoForTokenAuth.")
     }
+    
+    func testAuthenticatorInitHasCorrectProperties() {
+        //COME BACK TO THIS
```

Comment:
This was a note to myself because I started out trying to create a function that tested the init for the class, but realized pretty quick that the test I was working on was bad.  So I moved on with the intention of returning and missed the note while I was cleaning up the tests before creating the PR.

I removed this test because it's not doing much, and the properties being tested in the init are used throughout the rest of the tests.

Reply:
Can do.  Thanks for the suggestion